menas/ui/css/style.css

```diff
@@ -127,3 +127,15 @@ h5[id$=datasetDetailView--scheduleTimingTitle] {
   padding: 8px 2px;
   margin-top: 4px;
 }
+
+.sapMObjStatusError{
+  font-style: italic;
+}
+
+.sapMObjStatusWarning{
+  font-style: italic;
+}
+
+.sapMObjStatusInformation{
+  font-style: italic;
+}
```

Comment:
This could be simplified to
```css
.sapMObjStatusError, .sapMObjStatusWarning, .sapMObjStatusInformation{
  font-style: italic;
}
```

But even so, I don't think that it is ideal, because you could be inadvertently modifying style globally - it can have an adverse effect somewhere else by changing the style of these universal classes (which I believe you don't intend to do). 

Maybe it would be more fitting to find another way to introduce the change to the properties in dataset view only, of the top of my head, the options might be:
 - if you can, wrap the missing labels with a custom class (is this even possible with `FormElement` or `ObjectStatus`?) and define the style there, e.g. 
```css
.missingObjectStatus {
  font-style: italic;
}```

 - define a wrapper class higher in the form-container structure and limit the effect, as in
```css
.myDsView .sapMObjStatusError, .myDsView .sapMObjStatusWarning, .myDsView .sapMObjStatusInformation{
  font-style: italic;
}
```
  - if you had trouble doing this (too), perhaps using one of the generated ids (`__navigation0---datasetDetailView--info--Grid`?,  `__navigation0---datasetDetailView--info`?) could work as well, in the worst case scenario

Reply:
I would try the second option